src/applications/disability-benefits/all-claims/content/common.jsx

```diff
@@ -46,3 +47,17 @@ export const ContactCrisis = () => (
     </p>
   </>
 );
+
+export const bddAlertBegin = (
+  <>
+    <p className="vads-u-font-size--base">
+      You’ll need to upload your completed{' '}
+      <a href={DBQ_URL} target="_blank" rel="noreferrer">
+        Separation Health Assessment - Part A Self-Assessment
+      </a>{' '}
```

Comment:
Our a11y & content people have started recommending including text to let the user know that the link opens a page in a new tab.

Reply:
This content was reviewed by OCTO and this is the first I'm hearing of this so we'll keep this in mind for the future.